src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor(){
+        String[] color = new String[]{"Black", "Yellow", "Orange", "White", "Purple", "Blue", "Green", "Pink"};
+        Random random = new Random();
```

Comment:
The `Random` instance should be a class-level field, not a local variable in the method, to avoid creating a new instance every time the method is called.